src/logger.ts

```diff
@@ -63,7 +68,10 @@ export class ConsoleLogger extends OptionalLoggerImpl {
  */
 export const consoleLogger: Logger = {
   log(level: LogLevel, ...args: unknown[]): void {
-    console[level](...args);
+    // This really isn't the right place to put the log level tag, we should instead
+    // have a formatter that can be passed to a logger (including the consoleLogger)
+    // and we can then use that formatter regardless of where the log is going.
+    console[level](logLevelTag[level], ...args);
```

Comment:
I don't like this. `console.log` already outputs these with differently:

<img width="563" alt="Screen Shot 2022-03-22 at 13 54 24" src="https://user-images.githubusercontent.com/45845/159486552-8378b92d-bb36-48aa-bf7e-00a2999ff99e.png">
 
I guess what you want is for nodejs to output them differently. Maybe add a specific nodejs LogSink?

Reply:
ok try https://github.com/rocicorp/logger/pull/9 on for size. i bumped the version as part of that change, but did not publish a new package (wanted to wait for you to give the thumbs up).